roles/pulp/tasks/main.yaml

```diff
@@ -46,6 +76,75 @@
     port: 8080
     timeout: 300
 
+- name: Wait for Pulp API service to be accessible
```

Comment:
This code is failing to find the pulp-api port because the service isn't started until further down the file (https://github.com/theforeman/foreman-quadlet/pull/34/files#diff-6d531cd8425f805f75bfb941463407fb86c42731bd3036def1633d683def9cf0R124).

Line 67-77 are starting the old all-in-one and not the `pulp-api` service so this will never pass in it's current spot.